test/ui/QuerySummaryTest.ts

```diff
@@ -135,25 +135,25 @@ export function QuerySummaryTest() {
 
       it('onlyDisplaySearchTips allow to not render the results range', () => {
         test = Mock.optionsComponentSetup<QuerySummary, IQuerySummaryOptions>(QuerySummary, {
-          onlyDisplaySearchTips: false
+          // onlyDisplaySearchTips: false
```

Comment:
The same UT should still be working after the changes. It's generally a good sign that the changes we did were non-breaking :)

Reply:
Good point :)